dir.targets

```diff
@@ -10,6 +10,15 @@
   <Target Name="BuildAndTest" DependsOnTargets="Build;Test" />
   <Target Name="RebuildAndTest" DependsOnTargets="Rebuild;Test" />
   <Target Name="Test" />
+  
+  <ItemGroup Condition="'$(IsTestProject)' == 'true'" >
```

Comment:
I thought about that, however it's possible to be null/empty... so an "equals false" check would fail to work correctly when IsTestProject is unset.